lib/src/animate.dart

```diff
@@ -320,7 +324,9 @@ class _AnimateState extends State<Animate> with SingleTickerProviderStateMixin {
     for (EffectEntry entry in widget._entries) {
       child = entry.effect.build(context, child, _controller, entry);
     }
-    return reparent?.call(parent, child) ?? child;
+    Widget returnWidget = reparent?.call(parent, child) ?? child;
+    if (widget.useRepaintBoundary) return RepaintBoundary(child: returnWidget);
```

Comment:
instead of an additional return, I'd just add to returnWidget. I wonder if there's a better name than returnWidget also.